_posts/en/2021-09-29-krakow-riichi-open.markdown

```diff
@@ -0,0 +1,54 @@
+---
+title:  "Kraków Rīchi Open"
+date:   2021-09-29 17:00:00 +0100
+categories: tournaments
+lang: pl
+---
+
+{% capture fig_img %}
+![Kraków Rīchi Open]({{ "/assets/images/2021-09-29-krakow-riichi-open.png" | relative_url }})
+{% endcapture %}
+
+<figure>
+  {{ fig_img | markdownify | remove: "<p>" | remove: "</p>" }}
+</figure>
+
+In accordance to our previous declaration made when we have notified about
+creating an association, Krakow Chombo Club is proud to announce the first
+Riichi Mahjong tournament in Krakow! The tournament will be played with
```

Comment:
first open Riichi Mahjong

Reply:
niespójne - tu jest Riichi, w polskiej jest Rīchi

w ogóle chcemy używać `ī` jeśli we wszystkich innych materiałach jest `ii`? zdecydowałbym się na `ii` wszędzie żeby nie komplikowax